### DISCUSSION TITLE
time stamp on the stats chart?

### DISCUSSION THREAD

**Discussion by gavinsboyle** [sly9/VoyagerTelegramBot#34] [Q&A]
hi guys 

it looks like the time "pin" on the stats chart is the time it was initially pinned.
might it be possible to include somewhere on the chart the time of the last refresh as with multiple sequences running a night I am sometimes not 100% sure that the chart is current and that would be an easy way to check...?

thanks
gavin

**Comment by sly9** [comment]
This is a good idea, we can even include time stamps for the first and last image, so that we know the timespan of this sequence
